frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetailsActions.tsx

```diff
@@ -55,6 +61,8 @@ const PipelineDetailsActions: React.FC<PipelineDetailsActionsProps> = ({
           </DropdownItem>,
           <DropdownSeparator key="separator-create" />,
           <DropdownItem
+            isDisabled={!isPipelineSupported}
+            tooltip={PIPELINE_CREATE_RUN_TOOLTIP_ARGO_ERROR}
```

Comment:
this tooltip will not show. you have to use `aria-disabled` instead of isDisabled